lib/mspec/helpers/ruby_exe.rb

```diff
@@ -128,9 +135,17 @@ def ruby_exe(code = :not_given, opts = {})
     code = tmpfile
   end
 
+  exception = opts.fetch(:exception, true)
+
   begin
     platform_is_not :opal do
-      `#{ruby_cmd(code, opts)}`
+      output = `#{ruby_cmd(code, opts)}`
+
+      if !$?.success? && exception
+        raise "Ruby command failed. Exit status #{$?.exitstatus}"
```

Comment:
The error message should show the command that failed

Reply:
FWIW
```
$ ruby -e 'system "exit 3", exception: true'
Traceback (most recent call last):
	1: from -e:1:in `<main>'
-e:1:in `system': Command failed with exit 3: exit 3 (RuntimeError)
```
But I find it rather unclear.

So I'm thinking something like:
`"ruby_exe(#{command.inspect}) failed: {$?.inspect}"`

Reply:
Done